modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6220,6 +6220,7 @@ for-example,-if-the-public-virtual-host-is-www.helloworld.com-and-the-friendly-u
 for-example-x=For example, {0}.
 for-internal-use-only=For Internal Use Only
 for-locale-x-x-was-changed-to-x=For locale {0}, {1} changed to {2}.
+for-more-accurate-data,-require-user-athentication=For more accurate data, require user athentication
```

Comment:
@marcelabc I believe we should either add a period to the end of the sentence or start all words in capital letters.

In addition, is this comma really necessary (`for-more-accurate-data,`)? Never seen this before.

Reply:
>  believe we should either add a period to the end of the sentence or start all words in capital letters. 👍 

> In addition, is this comma really necessary (for-more-accurate-data,)? Never seen this before.
Answer: I searched in the *.properties files and I found examples that do the same pattern. The mockup asks for this comma in the sentence.

Reply:
Thanks @marcelabc